### PR TITLE
update react 15.3

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: desktop/package.json [hunk 1]
```diff
@@ -39,17 +39,17 @@
     "getenv": "0.6.0",
     "immutable": "3.8.1",
     "jsonfile": "2.3.1",
-    "lodash": "4.13.1",
-    "marked": "0.3.5",
-    "material-ui": "0.15.2",
-    "menubar": "5.0.0",
+    "lodash": "4.14.2",
+    "marked": "0.3.6",
+    "material-ui": "0.15.3",
```
Comment (chrisnojima) [inline desktop/package.json]: this is a bugfix release. dumb sheet looks fine

File: desktop/package.json [hunk 2]
```diff
@@ -39,17 +39,17 @@
...
+    "menubar": "5.1.0",
```
Comment (chrisnojima) [inline desktop/package.json]: this fixes the highlight area of the bar (finally) and not much else

File: desktop/package.json [hunk 3]
```diff
@@ -39,17 +39,17 @@
...
     "mkdirp": "0.5.1",
     "moment": "2.14.1",
     "purepack": "1.0.4",
     "qrcode-generator": "1.0.0",
-    "react": "15.2.1",
-    "react-addons-css-transition-group": "15.2.1",
-    "react-dom": "15.2.1",
+    "react": "15.3.0",
```
Comment (chrisnojima) [inline desktop/package.json]: now we can use PureComponent!

File: desktop/package.json [hunk 4]
```diff
@@ -58,42 +58,42 @@
...
+    "babel-preset-stage-2": "6.13.0",
+    "babel-runtime": "6.11.6",
     "chai": "3.5.0",
     "chai-as-promised": "5.3.0",
     "del": "2.2.1",
-    "devtron": "1.2.1",
+    "devtron": "1.3.0",
     "electron-download": "git://github.com/keybase/electron-download#5350b5a1bad8ba7bb8c7201abe59ae3a03c3d75d",
-    "electron-packager": "7.3.0",
+    "electron-packager": "7.5.1",
```
Comment (chrisnojima) [inline desktop/package.json]: electron actually got the npm name 'electron', this supports when this changes later

File: desktop/renderer/remote-component-loader.js [hunk 5]
```diff
@@ -154,7 +154,7 @@ class RemoteComponentLoader extends Component {
   render () {
     const Component = this.Component
     if (!this.state.loaded) {
-      return <div style={styles.loading}></div>
+      return <div style={styles.loading} />
```
Comment (chrisnojima) [inline desktop/renderer/remote-component-loader.js]: new eslint rules

**Comment by chrisnojima** [comment]
@keybase/react-hackers updated deps, nothing exciting other than PureComponent

**Comment by gabriel** [comment]
This updates material-ui? Electron is still 1.3.2... PR title is confusing

**Comment by chrisnojima** [comment]
Sorry it's react 153

File: desktop/package.json (hunk 2 above)
Comment (awendland) [inline desktop/package.json]: The icon is highlighting on selection (yay!) but it's not inverting color on highlight like the other icons in my menubar. Do we have a ticket for this already?
screen shot 2016-08-10 at 9 36 49 am

File: desktop/package.json [hunk 6]
```diff
@@ -33,23 +33,23 @@
...
     "framed-msgpack-rpc": "git://github.com/keybase/node-framed-msgpack-rpc#f3e1a3eb410e0bcde950bae14ceef7015c7315c8",
     "getenv": "0.6.0",
     "immutable": "3.8.1",
     "jsonfile": "2.3.1",
-    "lodash": "4.13.1",
-    "marked": "0.3.5",
-    "material-ui": "0.15.2",
-    "menubar": "5.0.0",
+    "lodash": "4.14.2",
+    "marked": "0.3.6",
```
Comment (awendland) [inline desktop/package.json]: Where is marked used in the codebase? Some quick grepping didn't turn up anything for me.

**Comment by awendland** [comment]
👍, see comments for potential ticketable items

File: desktop/package.json (hunk 2 above)
Comment (chrisnojima) [inline desktop/package.json]: this is an electron issue so we'd have to do it there. this is better than what happens w/ the version we have imho

File: desktop/package.json (hunk 6 above)
Comment (chrisnojima) [inline desktop/package.json]: it used to be used in the updater but we removed that code. i'll remove it

**Pull request merged**